syntax = "proto3";
package pl.canonical_message;

option go_package = "canonical";
// Try sending the message as a completely defined canonical message

message Canonical {
  int64 time_stamp = 1;
  double data1 = 2;
  int64 data2 = 3;
}

message CanonicalRepeatedColumn {
  repeated int64 time_stamp = 1;
  repeated double data1 = 2;
  repeated int64 data2 = 3;
}

message CanonincalStream {
  repeated Canonical data_stream = 1;
}
